### ISSUE TITLE
Android version of LocalizationKit

### ISSUE THREAD

**Issue by neokree** [willpowell8/LocalizationKit_iOS#19]
Hi,

I would like to know if you will port this library to Android in the future,
Because It would be very useful to have a single service for localizing apps for iOS and Android.

Please let me know,
Thank you

**Comment by willpowell8** [comment]
Yes this is my plan. I want to create an android version now that I have updated to version 2.0.0 which has inline localization tools. If you would like to help let me know.

**Comment by ventis07** [comment]
Hi, I sent you a mail regarding the Android version of this project. Will send it again but what help(s) do you need/require for the Android porting. Am sure @neokree and I could be of help as I wish to use a single codebase for over the air translation in my Android and iOS apps.

Expect your response.

**Comment by willpowell8** [comment]
Thank you for your email. I am working on an android version.

**Comment by ventis07** [comment]
I was wondering if this link could help https://github.com/mizutori/InAppTranslation_Android. Its a concept about Data Binding and thought you could use it for creating an android utilization of your Localization kit. The only difference is that the inline localization possible on iOS will not be possible
but every localization completed on the iOS will be bound to the android app. 

With this concept, the iOS acts as the main source of localization while the android client 
uses localization done. 

I hope i make some sense.

**Comment by ventis07** [comment]
Hi Will, 

I'd like to inquire on the progress of your android version for this project. Please how long have to you go?

Thank you.
